applications/incompressible_navier_stokes/poiseuille/application.h

```diff
@@ -476,17 +476,18 @@ class Application : public ApplicationBase<dim, Number>
     pp_data.output_data.time_control_data.is_active        = this->output_parameters.write;
     pp_data.output_data.time_control_data.start_time       = start_time;
     pp_data.output_data.time_control_data.trigger_interval = (end_time - start_time) / 10.0;
-    pp_data.output_data.directory                 = this->output_parameters.directory + "vtu/";
-    pp_data.output_data.filename                  = this->output_parameters.filename;
-    pp_data.output_data.write_vorticity           = true;
-    pp_data.output_data.write_divergence          = true;
-    pp_data.output_data.write_shear_rate          = true;
-    pp_data.output_data.write_velocity_magnitude  = true;
-    pp_data.output_data.write_vorticity_magnitude = true;
-    pp_data.output_data.write_processor_id        = true;
-    pp_data.output_data.write_q_criterion         = true;
-    pp_data.output_data.degree                    = this->param.degree_u;
-    pp_data.output_data.write_higher_order        = true;
+    pp_data.output_data.directory                      = this->output_parameters.directory + "vtu/";
+    pp_data.output_data.filename                       = this->output_parameters.filename;
+    pp_data.output_data.write_vorticity                = true;
+    pp_data.output_data.write_divergence               = true;
+    pp_data.output_data.write_shear_rate               = true;
+    pp_data.output_data.write_velocity_magnitude       = true;
+    pp_data.output_data.write_vorticity_magnitude      = true;
+    pp_data.output_data.write_wall_shear_stress_on_IDs = {dealii::numbers::invalid_boundary_id};
```

Comment:
We should implement it in a way such that this default initialization is not necessary.

Reply:
now, the default is: write on all  boundaries, if `write_wall_shear_stress == true`, and the user can specify a std::set of `boundary_id`s. I would like this more than just allowing a single one, since we might use the boundary ID more actual varying boundary conditions *on* the wall (thrombus growth, heated surface or whatever - some multiphysics application). In my mind this is just a bit more flexible.